sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ReactorHandlerProvider.java

```diff
@@ -61,7 +62,8 @@ ConnectionHandler createConnectionHandler(String connectionId, String hostname,
      * @param openTimeout Duration to wait for the session to open.
      * @return A new {@link SessionHandler}.
      */
-    SessionHandler createSessionHandler(String connectionId, String host, String sessionName, Duration openTimeout) {
+    public SessionHandler createSessionHandler(String connectionId, String host, String sessionName,
```

Comment:
All the method in this class should use `hostname` instead of `host` to be consistent with `ConnectionOptions` class.